src/openfl/text/TextField.hx

```diff
@@ -1365,16 +1353,20 @@ class TextField extends InteractiveObject {
 		
 	}
 	
-	inline function __ensureCaretVisible () {
+	function __ensureCaretVisible () {
 		
-		__ensureCaretVisibleNeeded = true;
+		if (!hasEventListener (Event.ENTER_FRAME)) {
+			
+			addEventListener (Event.ENTER_FRAME, __doEnsureCaretVisible);
+			
+		}
```

Comment:
As you have `removeEventListener` in one line, I would put this here in one line as well. Or put the below one in multiple lines. I would just make it consistent.

Reply:
I guess/hope we'll run the formatter on this codebase once and it'll equalize everything :)